app/models/scheme.rb

```diff
@@ -17,39 +17,37 @@ class Scheme < ApplicationRecord
   enum sensitive: SENSITIVE, _suffix: true
 
   REGISTERED_UNDER_CARE_ACT = {
-    "No": 1,
     "Yes – registered care home providing nursing care": 4,
     "Yes – registered care home providing personal care": 3,
     "Yes – part registered as a care home": 2,
+    "No": 1,
   }.freeze
 
   enum registered_under_care_act: REGISTERED_UNDER_CARE_ACT
 
   SCHEME_TYPE = {
-    "Missing": 0,
-    "Foyer": 4,
     "Direct Access Hostel": 5,
-    "Other Supported Housing": 6,
+    "Foyer": 4,
     "Housing for older people": 7,
+    "Other Supported Housing": 6,
+    "Missing": 0,
```

Comment:
Why do we explicitly need a missing option? Is this equivalent to something other than null?

Reply:
It was in the list of types in the doc, I presumed this would required on import. Will be happy to remove if deemed redundant.

Reply:
@smeny what do you think?